gamestonk_terminal/stocks/discovery/seeking_alpha_view.py

```diff
@@ -149,3 +149,35 @@ def news(news_type: str, article_id: int, num: int, start_date: datetime, export
             news_type,
             df_articles,
         )
+
+
+def display_news(news_type: str = "Top-News", num: int = 5, export: str = ""):
+    """Display news. [Source: SeekingAllpha]
+
+    Parameters
+    ----------
+    news_type : str
+        From: Top-News, On-The-Move, Market-Pulse, Notable-Calls, Buybacks, Commodities, Crypto, Issuance, Global,
+        Guidance, IPOs, SPACs, Politics, M-A, Consumer, Energy, Financials, Healthcare, MLPs, REITs, Technology
+    num : int
+        Number of news to display
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+    l_news = seeking_alpha_model.get_news(news_type, num)
```

Comment:
With mypy you can have more descriptive names. There is really no need at type hinting in the name.

You can use the explicit syntax if you like:

```
news: List[dict] =  seeking_alpha_model.get_news(news_type, num)
```

Reply:
I'm used to be more explicit, it helps me code faster on the notebook.

Then when I copy code across our structure sometimes I miss changing these :) Will fix.